provider/ec2/environ.go

```diff
@@ -1153,7 +1153,7 @@ func (e *environ) subnetsForVPC(ctx context.ProviderCallContext) (resp *ec2.Subn
 // AdoptResources is part of the Environ interface.
 func (e *environ) AdoptResources(ctx context.ProviderCallContext, controllerUUID string, fromVersion version.Number) error {
 	// Gather resource ids for instances, volumes and security groups tagged with this model.
-	instances, err := e.AllInstances(ctx)
+	instances, err := e.AllRunningInstances(ctx)
```

Comment:
Eventually this should definitely be AllInstances, but I guess you're still doing the mechanical transformation in this pass?

Reply:
Yes, I'd rather we changed the calls when the need actually arises. However, if you are sure that AdoptResources should be dealing with all cloud instances irrespective of their state (I guess it does since this is essentially what we need for migration), I am happy to do a follow-up.